src/SIM/SIMbase.C

```diff
@@ -1556,9 +1556,10 @@ bool SIMbase::solutionNorms (const TimeDomain& time,
   norm->initIntegration(nIntGP,nBouGP);
 
   // Number of recovered solution components
-  if (!this->fieldProjections())
+  size_t nNodes = this->getNoNodes(1);
+  if (!this->fieldProjections() && nNodes > 0)
```

Comment:
switch the order in the logic expression, and we save one `fieldProjections()` invokation in the event of `nNodes=0`

Reply:
changed.